jobs/release-monitoring/discovery/middleware-monitoring.yaml

```diff
@@ -8,9 +8,9 @@
       - timed: '@hourly'
     parameters:
       - string:
-          name: 'manifestVar'
+          name: 'productVersionVar'
           default: 'middleware_monitoring_operator_release_tag'
```

Comment:
I'd have expected this to releaseTagVar since it's a release tag being updated?

Reply:
having it as a releaseTag here wouldn't have worked with the current Jenkinsfile script. But will update to use with your changes in this PR https://github.com/integr8ly/ci-cd/pull/75/files